ts/components/services/ContactPreferencesToggles/index.tsx

```diff
@@ -108,6 +109,7 @@ const ContactPreferencesToggle: React.FC<Props> = (props: Props) => {
       <PreferenceToggleRow
         label={I18n.t("services.serviceIsEnabled")}
         onPress={(value: boolean) => onValueChange(value, "inbox")}
+        disabled={props.isSpecialService}
```

Comment:
Could you leave a comment describing why toggle must be disabled when the service is special?

Reply:
@Undermaken Added in 4877c8d